packages/tools/fetch-tool/src/fluidFetchInit.ts

```diff
@@ -115,7 +115,7 @@ async function initializeR11s(server: string, pathname: string, r11sResolvedUrl:
     console.log(`Connecting to r11s: tenantId=${tenantId} id:${documentId}`);
     const tokenProvider = new r11s.DefaultTokenProvider(paramJWT);
     const r11sDocumentServiceFactory = new r11s.RouterliciousDocumentServiceFactory(tokenProvider);
-    return r11sDocumentServiceFactory.createDocumentService(r11sResolvedUrl);
+    return r11sDocumentServiceFactory.createDocumentService(r11sResolvedUrl, undefined, false);
```

Comment:
Here, and in other places: It would be useful to have these kinds of comments. This makes code more readable:

```
createDocumentService(
   r11sResolvedUrl,
   undefined, // <Name of argument>
   false); // isSummarizer
```